src/layouts/SiteLayout.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { StaticQuery, graphql } from "gatsby"
+import { Box } from "rebass"
+import { maxWidth, MaxWidthProps } from "styled-system"
+import Header from "../components/Header"
+import styled from "styled-components"
+
+const PageContent = styled(Box)<MaxWidthProps>`
+  ${maxWidth};
+`
+
+const Layout = ({ children }) => (
+  <StaticQuery
+    query={graphql`
+      query SiteTitleQuery {
+        site {
+          siteMetadata {
+            title
+          }
+        }
+      }
+    `}
+    render={data => (
+      <>
+        <Header siteTitle={data.site.siteMetadata.title} />
+        <PageContent m="0 auto" maxWidth={["560px"]}>
+          <Box p="0px 1.0875rem 1.45rem">
```

Comment:
`p="0px 1.0875rem 1.45rem"`: I'm just replacing the padding prop from the gatsby starter with a string literal to be used as css- No reliance on a theme here.